packages/typespec-ts/src/framework/hooks/binder.ts

```diff
@@ -39,15 +45,18 @@ export interface Binder {
 
 class BinderImp implements Binder {
   private declarations = new Map<unknown, DeclarationInfo>();
+  private references = new Map<unknown, Set<SourceFile>>();
```

Comment:
Adding this map to track things that are used, this is helpful to cleanup unused static-helpers